crates/genesis/src/rollup.rs

```diff
@@ -140,6 +140,44 @@ pub struct RollupConfig {
     pub da_challenge_address: Option<Address>,
 }
 
+#[cfg(any(test, feature = "arbitrary"))]
+impl<'a> arbitrary::Arbitrary<'a> for RollupConfig {
+    fn arbitrary(u: &mut arbitrary::Unstructured<'a>) -> arbitrary::Result<Self> {
+        Ok(Self {
+            genesis: ChainGenesis::arbitrary(u)?,
+            block_time: u.arbitrary()?,
+            max_sequencer_drift: u.arbitrary()?,
+            seq_window_size: u.arbitrary()?,
+            channel_timeout: u.arbitrary()?,
+            granite_channel_timeout: u.arbitrary()?,
+            l1_chain_id: u.arbitrary()?,
+            l2_chain_id: u.arbitrary()?,
+            base_fee_params: BaseFeeParams {
```

Comment:
could we derive this if this were arbitrary?

Reply:
Yes

Reply:
we can easily add

Reply:
I'm happy to take that on if you'd like

Reply:
yes please

perhaps this should use the regular op or eth params always

Reply:
Oh good point... will put up an issue for this so as to not block.